### PR TITLE
[fix docs compiler warnings] Glutton pallet

### PR DESCRIPTION
**Pull request by sacha-l**
This PR fixes 7 missing doc warnings in the Glutton pallet

### CODE REVIEW

File: frame/glutton/src/lib.rs [hunk 1]
```diff
@@ -75,11 +75,20 @@ pub mod pallet {
...
 		/// The pallet has been (re)initialized.
-		PalletInitialized { reinit: bool },
+		PalletInitialized {
+			/// The pallet initialization state.
+			reinit: bool,
+		},
 		/// The computation limit has been updated.
-		ComputationLimitSet { compute: FixedU64 },
+		ComputationLimitSet {
+			/// The computation limit.
```
Comment (joepetrowski) [inline frame/glutton/src/lib.rs:84]: Annoying that we say "compute" everywhere else and "computation" here.

**Review by joepetrowski — APPROVED** [review]

**Review by NachoPal — APPROVED** [review]

File: frame/glutton/src/lib.rs [hunk 2]
```diff
@@ -75,11 +75,20 @@ pub mod pallet {
 	#[pallet::generate_deposit(pub(super) fn deposit_event)]
 	pub enum Event {
 		/// The pallet has been (re)initialized.
-		PalletInitialized { reinit: bool },
+		PalletInitialized {
+			/// The pallet initialization state.
```
Comment (ggwpez) [inline frame/glutton/src/lib.rs:79]:
```suggestion
			/// Whether the pallet has already been re-initialized.
```

**Review by ggwpez — APPROVED** [review]

**Comment by ggwpez** [comment]
Please also add `#![deny(missing_docs)]`, otherwise this will get out-dated.

File: frame/glutton/src/lib.rs (hunk 1 above)
Comment (sacha-l) [reply frame/glutton/src/lib.rs:84]: Yep agree. Right now the goal is to fix the warnings. We may want to do a pass on renaming some the variables to be more accurate in a follow-up. For this one, it makes more sense to use the term "computation" over "compute" imo.

**Comment by sacha-l** [comment]
bot merge

**Pull request merged**